### PR TITLE
Add dashboard icon face.

### PR DESCRIPTION
**Pull request by ricardoricho**
This is for #332

### CODE REVIEW

**Review by jcs090218 — APPROVED** [review]
LGTM! Thank you!

File: dashboard-widgets.el [hunk 1]
```diff
@@ -251,12 +251,6 @@ Example:
   :type '(function string)
   :group 'dashboard)
 
-(defcustom dashboard-footer
-  (nth (random (1- (1+ (length dashboard-footer-messages)))) dashboard-footer-messages)
-  "A footer with some short message."
-  :type 'string
-  :group 'dashboard)
```
Comment (jcs090218) [inline dashboard-widgets.el:258]: Why are we removing the random footer? 🤔

**Review by jcs090218 — COMMENTED** [review]
I've got a closer look. I think you have accidentally removed `dashboard-footer` (not the face).

File: dashboard-widgets.el (hunk 1 above)
Comment (ricardoricho) [reply dashboard-widgets.el:258]: We are not using that custom variable. The `dashboard-insert-footer` function uses `dashboard-random-footer` to get the value.  The `dashboard-footer-message` list is used to customize the text, from the README:
```emacs-lisp
(setq dashboard-footer-messages '("Dashboard is pretty cool!"))
```

Comment (jcs090218) [reply dashboard-widgets.el:258]: Good catch! Now sure why we still have this variable in the first place. 🤔

**Comment by jcs090218** [comment]
Thank you! Merged!

**Comment by jcs090218** [comment]
Just to let you know, I have reverted the `propertize`  in `dashboard-insert-footer` function (22786237e16cfeae33f07ae9c5eeaf061408579a) since propertize will make the icon undisplayable. 🤔 

Before changes:



After changes:

**Pull request merged**